### ISSUE TITLE
[Desktop] Investigate uncategorized remote connection error: "Cannot connect to Jupyter Lab Terminal. Please try again."

### ISSUE THREAD

**Issue by sevillal** [microsoft/vscode-tools-for-ai#2290]
```[tasklist]
### Tasks
- [ ] https://github.com/microsoft/vscode-tools-for-ai/issues/2291
```

**Comment by sevillal** [comment]
###Status
Fix merged available in **v0.46.0** (a week ago).

Instances of the error in v0.46.0 (in a week span): 4
Instances of the error in v0.44.1 (in a week span): 68

95% improvement.

**Comment by sevillal** [comment]
Closing for now, since it the error count decreased after the fix.
